pkg/semver/semver.go

```diff
@@ -280,6 +280,9 @@ func Parse(s string) (Version, error) {
 	if re.Match([]byte(parts[0])) {
 		prefix = re.FindString(parts[0])
 		parts[0] = strings.Replace(parts[0], prefix, "", 1)
+		if len(parts[0]) < 1 {
```

Comment:
This would break API compatibility wouldn't it? We're on `v0.0.*` so it doesn't matter so much.

This line checks for versions like `v.3.2` or something? Where first a prefix (non semver) is expected, but then after the removal of the prefix nothing is left?